.travis.yml

```diff
@@ -1,31 +1,26 @@
-# See https://github.com/silverstripe-labs/silverstripe-travis-support for setup details
+# See https://github.com/silverstripe/silverstripe-travis-support for setup details
 
 sudo: false
 
 language: php
 
 php:
-  - 5.3
-  - 5.4
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
 
 env:
-  - DB=MYSQL CORE_RELEASE=3.2
+  - DB=MYSQL CORE_RELEASE=master
```

Comment:
Updated, I've rebased and fixed the merge conflicts (some strange stuff in that last commit...!)